### ISSUE TITLE
BUG:  pyinstaller fails with python 3.8, numpy 1.18

### ISSUE THREAD

**Issue by teddcp** [numpy/numpy#15299]
hi all,

i am trying to make an exe file using pyinstaller. while clicking on the exe file, its giving me fatal error with stating 'unable to open the exe'. So  i checked in cmd line , there is the issue with numpy.
Then i uninstalled and installed it again, nut still the issue is there.

### Reproducing code example:

After opeing the exe file, error is :

![image](https://user-images.githubusercontent.com/56304523/72061528-16917400-32fc-11ea-85b3-cbbd3cb6ccd3.png)

while making the exe file using pyinstaller command : 

![image](https://user-images.githubusercontent.com/56304523/72061715-75ef8400-32fc-11ea-991d-330995fb2356.png)

### Error message:

54028 INFO: Looking for dynamic libraries
55761 WARNING: lib not found: libopenblas.SVHFG5YE3RK3Z27NVFUDAPL2O3W6IMXW.gfortran-win32.dll dependency of c:\users\debasis.panda\appdata\local\programs\python\python38-32\lib\site-packages\numpy\core\_multiarray_umath.cp38-win32.pyd
56379 WARNING: lib not found: libopenblas.SVHFG5YE3RK3Z27NVFUDAPL2O3W6IMXW.gfortran-win32.dll dependency of c:\users\debasis.panda\appdata\local\programs\python\python38-32\lib\site-packages\numpy\linalg\_umath_linalg.cp38-win32.pyd
56450 WARNING: lib not found: libopenblas.SVHFG5YE3RK3Z27NVFUDAPL2O3W6IMXW.gfortran-win32.dll dependency of c:\users\debasis.panda\appdata\local\programs\python\python38-32\lib\site-packages\numpy\linalg\lapack_lite.cp38-win32.pyd

### Numpy/Python version information:
python - 3.8
numpy -1.18.1

**Comment by mattip** [comment]
This is likely an issue with pyinstaller not copying the dlls from python into the final installation, maybe related to the new python/numpy versions. Maybe related to gh-14807. 
It seems pyinstaller changed the hook for numpy in pyinstaller/pyinstaller#4593, you might have more luck if you try a HEAD version of pyinstaller.

**Comment by h6197627** [comment]
I think it is Python 3.8 problem - somehow it does not respect PATH env variable.
I am using Numpy with Intel MKL for Windows taken from https://www.lfd.uci.edu/~gohlke/pythonlibs/#numpy. It comes with whole distribution of MKL library in **DLLs** folder packaged to Numpy wheel. Considering that I am using MKL from C++ code as well, I am having standalone installation, so I prefer to delete packaged version and to use standalone added to PATH env variable.
What I found right now, is that in Python 3.8 I can't do it anymore - it does not respect my PATH variable settings and importing Numpy results in error as for topic starter.
I tried to uninstall Python 3.8.1 and install Python 3.7.6 and Numpy 1.18.1 is working as expected successfully loading my standalone MKL dlls.
Also with Python 3.8.1 I tried to move MKL libraries into C:\Windows\System32 directory, that way it also successfully imports Numpy. I guess it looks into default directories only, without using user's custom settings.
